tests/tests/atmaero_control_p8

```diff
@@ -104,7 +104,6 @@ export USE_MERRA2=.true.
 export IAER=1011
 export NSTF_NAME=2,0,0,0,0
 
-export LHEATSTRG=.true.
```

Comment:
DT_INNER has a default value (from default_vars) of 1800 (because it was set to DT_ATMOS), which means it will still be set as 1800 unless you again change it in the test itself, where DT_ATMOS=720.

Reply:
My understanding is that we don't have any test has DT_ATMOS=1800s, I'd suggest remove this line. Also I think the DT_INNER can't be larger than the atmosphere time step.

Reply:
If 1800 is no longer the default value (used for the ``control`` test for ATM), then we should change it in default_vars to whatever ``control`` uses. 

But anytime DT_ATMOS changes in a test, we need to also then add a line to change DT_INNER, otherwise it will remain whatever the default DT_ATMOS is set to.